resources/platforms.edn

```diff
@@ -30,6 +30,15 @@
  ;; we track is available at:
  ;; https://docs.datastax.com/en/landing_page/doc/landing_page/supportedPlatforms.html#supportedPlatforms__dse-supported-platforms
 
+ "5.1.0" {"debian" ["8.x" "9.x"]
+          "ubuntu" ["12.04" "14.04" "16.04" "18.04"]
+          "red hat enterprise" ["6.7.x" "6.8.x" "6.9.x" "6.10.x" "7.1.x" "7.2.x"
+                                "7.3.x" "7.4.x" "7.5.x" "7.6.x" "7.7.x"]
+          "oracle" ["6.9.x" "7.4.x" "7.6.x"]
+          "centos" ["6.7.x" "6.8.x" "6.9.x" "6.10.x" "7.1.x" "7.2.x" "7.3.x"
+                    "7.4.x" "7.5.x" "7.6.x" "7.7.x"]
```

Comment:
I've actually seen emails flying around about the official support of centos and redhat 7.8. Any idea if we need to add them here?

Reply:
We should do this in it's own ticket.